packages/turbo-workspaces/jest.config.js

```diff
@@ -10,8 +10,8 @@ module.exports = {
   verbose: true,
   coverageThreshold: {
     global: {
-      branches: 86,
-      functions: 91,
+      branches: 83,
+      functions: 87,
```

Comment:
😲

Reply:
I added some more specific error handling for consumption by create-turbo, I'll go back and patch it up don't worry.